packages/x-utils/tsconfig.json

```diff
@@ -2,18 +2,13 @@
   "compilerOptions": {
     "target": "es2019",
     "module": "commonjs",
-    "outDir": "./dist/cjs",
-    "declaration": true,
-    "declarationDir": "./types",
     "moduleResolution": "node",
     "strict": true,
     "esModuleInterop": true,
     "types": ["jest"],
     "lib": ["esnext"],
-    "sourceMap": true,
-    "inlineSources": true,
     "importHelpers": true
   },
   "include": ["src/**/*.ts"],
-  "exclude": ["node_modules", "**/__tests__/**/*"]
+  "exclude": ["node_modules"]
```

Comment:
I've removed the `tests` files from being `excluded` as this `tsconfig.json` is now the one used by the `IDE`/`compilation` process and, as such, needs to include those files.